go/systests/tracking_test.go

```diff
@@ -0,0 +1,143 @@
+// Copyright 2015 Keybase, Inc. All rights reserved. Use of
+// this source code is governed by the included BSD license.
+
+package systests
+
+import (
+	"testing"
+
+	"github.com/keybase/client/go/client"
+	"github.com/keybase/client/go/engine"
+	"github.com/keybase/client/go/kbtest"
+	"github.com/keybase/client/go/libkb"
+	keybase1 "github.com/keybase/client/go/protocol"
+	"github.com/keybase/client/go/service"
+	rpc "github.com/keybase/go-framed-msgpack-rpc"
+	context "golang.org/x/net/context"
+)
+
+type trackingNotifyHandler struct {
+	trackingCh chan keybase1.UID
+	errCh      chan error
+}
+
+func newTrackingNotifyHandler() *trackingNotifyHandler {
+	return &trackingNotifyHandler{
+		trackingCh: make(chan keybase1.UID),
+		errCh:      make(chan error),
+	}
+}
+
+func (h *trackingNotifyHandler) TrackingChanged(_ context.Context, uid keybase1.UID) error {
+	h.trackingCh <- uid
+	return nil
+}
+
+func TestTrackingNotifications(t *testing.T) {
+	tc := setupTest(t, "signup")
+
+	// Hack the various portions of the service that aren't
+	// properly contextified.
```

Comment:
Add back in the `	libkb.G.LocalDb = nil` line. Also take a look at using the same Cmd* interfaces that the user_test.go does.